setup/index.js

```diff
@@ -69,6 +69,9 @@ async function setup() {
     `HYGEN_OVERWRITE=1 yarn hygen setup splashscreen --displayName ${displayName} --bundleIdentifer ${bundleIdentifer}`,
   );
 
+  console.log('\n🥜  Installing ios pods...');
+  execSync('cd ios && pod install', { cwd: rootDirectory });
```

Comment:
Should we switch this?

```suggestion
  execSync('pod install', { cwd: 'ios' });
```

Assuming we've ran `bundle install` it might be even better to use `bundle exec pod install`: https://bundler.io/man/bundle-exec.1.html

Reply:
Good call! I'm closing this PR in favor of #57 - but I'll make the change there.